app/models/user.rb

```diff
@@ -7,4 +7,7 @@ class User < ActiveRecord::Base
   has_many :missions, dependent: :destroy
   has_many :mission_comments, class_name: Mission::Comment.name
   has_many :supervisions, class_name: Mission.name, inverse_of: :supervisor
+
+  has_many :friendships, dependent: :destroy, inverse_of: :user
+  has_many :friends, through: :friendships, class_name: "User"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.